docs/src/browsers.md

```diff
@@ -95,6 +96,22 @@ var chromium = playwright.Chromium;
 var browser = await chromium.LaunchAsync(new BrowserTypeLaunchOptions { Channel = "chrome" });
 ```
 
+### Installing Google Chrome & Microsoft Edge
+
+If stock Google Chrome or Microsoft Edge is not available on your machine, you can install
+them using Playwright command line tool:
+
+```bash
```

Comment:
This should include the snippets for the other language ports

Reply:
good point!